app/javascript/components/organizations/organization_detail.jsx

```diff
@@ -48,23 +48,23 @@ class OrganizationDetail extends React.Component {
                 </Box>
               </TypoGraphy>
             </Grid>
-          
+
           </Grid>
-          
+
           <br/>
           <br/>
-          
+
           <Grid container direction="row">
             <Grid container direction="row" justify="center" alignItems="stretch">
-              {organization.approved_events.map((event, i) =>
+              {approved_events.map((event, i) =>
```

Comment:
Try `({event: {name, address, start_time, end_time, first_imge_url}, i})` to get rid of repetitions of `event`. Then use it wisely on line 61-62 (`name={name}`)